### PR TITLE
Set Identity Module

### PR DESCRIPTION
**Pull request by PiZZAD0X**
- Sets random faces and voices from arrays in setting file
- Sets for units of each side
- Has unit and group excludes

### CODE REVIEW

File: modules/set_identity/settings.sqf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//This module does things
```
Comment (EM-Creations) [inline modules/set_identity/settings.sqf:1]: Can we just make this comment a wee bit more useful or remove it as the settings file is often the first place people look when wanting to use a module?

**Comment by EM-Creations** [comment]
Doesn't appear to be working when I've tested it, I've notified @PiZZAD0X of this.

**Comment by PiZZAD0X** [comment]
yup needs a frame wait and a remoteexec i will change

**Comment by EM-Creations** [comment]
> yup needs a frame wait and a remoteexec i will change

Thanks :)

**Comment by EM-Creations** [comment]
I've fixed it and merged in [this PR](https://github.com/EM-Creations/Jackalope-Framework-ArmA-3/pull/39)

The problem was actually the "exitWith" statement in the loop in init.sqf, it was short-circuiting the forEach execution so only the first array was being checked against.

**Pull request closed without merging**